src/views/Swap/Swap.vue

```diff
@@ -864,9 +864,28 @@ export default {
       return this.selectedQuote?.receiveFee
     },
     maxFee() {
-      const selectedSpeed = this.selectedFee[this.assetChain]
-      const fee = this.maxSwapFees[this.assetChain]?.[selectedSpeed]
-      return fee ? currencyToUnit(cryptoassets[this.assetChain], fee) : BN(0)
+      try {
+        const selectedSpeed = this.selectedFee[this.assetChain]
+        const fee = this.maxSwapFees[this.assetChain]?.[selectedSpeed] || 0
+
+        const getExtraAmountToExtractFromBalance =
+          this.selectedQuoteProvider?.getExtraAmountToExtractFromBalance
+        let extraAmountToExtractFromBalance = 0
+        if (getExtraAmountToExtractFromBalance) {
+          extraAmountToExtractFromBalance = getExtraAmountToExtractFromBalance()
+        }
+
+        const totalFees = currencyToUnit(cryptoassets[this.assetChain], fee).plus(
```

Comment:
Just check if we need to apply the same formula that we have inside the balance or something similar `this.maxFee.plus(extraAmountToExtractFromBalance)).times(1.5)`